src/inference/src/threading/ie_executor_manager.cpp

```diff
@@ -29,6 +32,10 @@ class ExecutorManagerImpl : public ExecutorManager {
 
 }  // namespace
 
+ExecutorManagerImpl::~ExecutorManagerImpl() {
+    init.blocking_terminate();
```

Comment:
is it needed now?

Reply:
Yes, it is required. Without it, TBB threads continue to work.
I agree, the user should manage the TBB threads himself. Maybe we should provide a configuration flag for it?
Anyway this is a draft PR to test that our approach works.